cirq-google/cirq_google/engine/virtual_engine_factory_test.py

```diff
@@ -175,3 +175,20 @@ def test_create_from_proto_no_qubits():
         _ = factory.create_noiseless_virtual_engine_from_device(
             'sycamore', cirq.UNCONSTRAINED_DEVICE
         )
+
+
+def test_create_noisy_virtual_engine_from_processor_id_and_simulator_class():
+    for processor_id in ["rainbow", "weber"]:
+        simulator_class = cirq.Simulator
```

Comment:
nit: I would just pass in cirq.Simulator directly to simulator_class eliminating one line of code.